47.md

```diff
@@ -492,6 +574,29 @@ Notification:
 }
 ```
 
+### `hold_invoice_accepted`
+
+Description: Sent when a payer accepts (locks in) a hold invoice.
```

Comment:
```suggestion
Description: Sent when a payer accepts (locks in) a hold invoice. To avoid locking up funds in channels the hold invoice SHOULD be settled or canceled within a few minutes of receiving this event.
```